recipes/seqcluster/meta.yaml

```diff
@@ -26,7 +26,6 @@ requirements:
     - pandas
     - pybedtools
     - progressbar
-    - matplotlib
```

Comment:
Seems that matplotlib is used in `report.py`

Reply:
not needed any more. I removed the HTML function to work only with the sqlite output.

Reply:
The early build failure is now gone with #6555 being merged by @daler.
Now the build test failed since `report.py` still imports from `matplotlib`:
```python
from matplotlib import pyplot as plt
plt.ioff()
```